custom_components/plugwise/binary_sensor.py

```diff
@@ -174,9 +178,15 @@ async def async_setup_entry_gateway(hass, config_entry, async_add_entities):
 class GwBinarySensor(SmileSensor, BinarySensorEntity):
     """Representation of a Plugwise binary_sensor."""
 
-    def __init__(self, api, coordinator, name, dev_id, binary_sensor, model):
+    def __init__(
+        self, api, coordinator, name, dev_id, enabled_default, binary_sensor, model
+    ):
         """Set up the Plugwise API."""
-        super().__init__(api, coordinator, name, dev_id, binary_sensor)
+        self._enabled_default = enabled_default, True
+
+        super().__init__(
+            api, coordinator, name, dev_id, self._enabled_default, binary_sensor
+        )
```

Comment:
`self._enabled_default` will be a `Tuple` and is passed into `super()`. To me that does not seems to be right.

Reply:
True, wrong copy/paste remnant (well spotted)